### PR TITLE
[MRG] Restyling

### PR DESCRIPTION
**Pull request by annapasca**
- added all function to import data in `import_data.py`
- relative import
- modified docstring for Interfaces
- added private functions

### CODE REVIEW

**Comment by annapasca** [comment]
@davidmeunier79 @dmalt @jasmainak if for all of you is ok we can merge it

**Comment by jasmainak** [comment]
@annapasca can you make `codecov` happy first? Your PR is impacting coverage quite a bit.

**Comment by annapasca** [comment]
Hi @davidmeunier79 !
Travis gave me this error
```

$ flake8 --count ephypype
ephypype/gather/gather_conmats.py:22:7: W605 invalid escape sequence '\d'
1
```

**Comment by davidmeunier79** [comment]
> Hi @davidmeunier79 !
> Travis gave me this error
> 
> ```
> 
> $ flake8 --count ephypype
> ephypype/gather/gather_conmats.py:22:7: W605 invalid escape sequence '\d'
> 1
> ```
Looking for this error  I only found this discussion, where they solved the problem with adding 

**r**  before the '
so have a try with this
return [atoi(c) for c in re.split(r'(\d+)', text)]

**Comment by annapasca** [comment]
> return [atoi(c) for c in re.split(r'(\d+)', text)]

Thks! I'll try now!

**Comment by annapasca** [comment]
> > Hi @davidmeunier79 !
> > Travis gave me this error
> > ```
> > 
> > $ flake8 --count ephypype
> > ephypype/gather/gather_conmats.py:22:7: W605 invalid escape sequence '\d'
> > 1
> > ```
> 
> Looking for this error I only found [this discussion](https://github.com/PyCQA/pycodestyle/issues/766) , where they solved the problem with adding **r** before the **'**, so have a try with this:
> 
> `return [atoi(c) for c in re.split(r'(\d+)', text)] `

It worked! thks!!! :)

File: doc/generated/ephypype.pipelines.create_pipeline_time_series_to_spectral_connectivity.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ephypype.pipelines.create\_pipeline\_time\_series\_to\_spectral\_connectivity
+=============================================================================
+
+.. currentmodule:: ephypype.pipelines
+
+.. autofunction:: create_pipeline_time_series_to_spectral_connectivity
```
Comment (jasmainak) [inline doc/generated/ephypype.pipelines.create_pipeline_time_series_to_spectral_connectivity.rst]: This file shouldn't be committed ... it's generated automatically by sphinx gallery

**Comment by jasmainak** [comment]
@annapasca feel free to merge when you've fixed my last comment

**Comment by annapasca** [comment]
> @annapasca feel free to merge when you've fixed my last comment

ok! I accidentally pushed the .rst file :)

**Comment by annapasca** [comment]
> > @annapasca feel free to merge when you've fixed my last comment
> 
> ok! I accidentally pushed the .rst file :)

Can I merge by myself?

**Comment by jasmainak** [comment]
> Can I merge by myself?

Let me do it :) Probably always a good idea for someone else to merge.

**Comment by jasmainak** [comment]
Thanks for the PR and the fixes !

**Comment by annapasca** [comment]
> Thanks for the PR and the fixes !

thks to you! :)

**Pull request merged**